content/ja/handbook/marketing/blog/git-guide.md

```
@@ -0,0 +1,103 @@
+**複数の人が作業している**フィーチャーブランチにはマージワークフローを使用してください。MR をマスターにマージする際は、組み込みの[スカッシュとマージ](https://docs.gitlab.com/ee/user/project/merge_requests/squash_and_merge.html)機能を使用して、クリーンなアトミックでスカッシュされたコミットのみがマスターに到達するようにしてください。
+
+```console
+git checkout 0000-branch-name
+git fetch
+git merge origin/master
+git push origin  0000-branch-name
+```
+
+### リベースとは何ですか？なぜ気にする必要があるのですか？
+
+マージリクエストにしばらく作業していて多くの変更をコミットした場合、他の人が Web サイトのメインブランチに加えた変更を反映するために MR をリベース（更新）する必要があるかもしれません。これにより、[マージコンフリクト](#how-to-resolve-merge-conflicts)（他の誰かの変更があなたの変更と競合し、Git がどちらを受け入れるかわからない場合）や、古い MR に反映されていない技術的変更によるパイプラインの失敗を防ぐことができます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)